### PR TITLE
Add demo generate-data command [CLC-206]

### PR DESCRIPTION
**Pull request by mtyazici**

### CODE REVIEW

File: internal/sse/event.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	normalizedMsg := bytes.Replace(msg, []byte("\n\r"), []byte("\n"), -1)
+	msgHeaders := bytes.Split(normalizedMsg, []byte("\n"))
+	for _, line := range msgHeaders {
+		// Implementation spec is taken from:
+		// https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events#fields
+		switch headerName(line) {
+		case headerData:
+			tmp.Data = append(tmp.Data, dataValue(line)...)
+			// append new line after each data line per spec
+			tmp.Data = append(tmp.Data, []byte("\n")...)
```
Comment (yuce) [inline internal/sse/event.go:35]:
```suggestion
			tmp.Data = append(tmp.Data, '\n')
```

**Review by yuce — CHANGES_REQUESTED** [review]
Some ideas...

**Review by yuce — DISMISSED** [review]
Looks good!

File: base/commands/demo/demo_generate_data.go [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+	keyVals, err := keyValMap(ec)
+	if err != nil {
+		return err
+	}
+	ctx, cancel := signal.NotifyContext(ctx, os.Interrupt, os.Kill)
+	defer cancel()
+	chv, stop, err := ec.ExecuteBlocking(ctx, func(ctx context.Context, sp clc.Spinner) (any, error) {
+		sp.SetText(fmt.Sprintf("Generating wikipedia stream events: %s", name))
+		ch := generator.Stream(ctx)
+		return ch, nil
```
[the remark this answers is not in the document]
Comment (mtyazici) [reply base/commands/demo/demo_generate_data.go]: I removed the ExecuteBlocking statement

**Review by yuce — APPROVED** [review]
Looks good!

**Pull request merged**